modules/devkit-support/src/test/java/org/mule/security/oauth/FetchAccessTokenTestCase.java

```diff
@@ -46,7 +46,7 @@ protected void doSetUp() throws Exception
         this.objectStore = new InMemoryObjectStore<Serializable>();
 
         this.event = getTestEvent("");
-        event.getMessage().setProperty("state", "<<MULE_EVENT_ID=whatever>>", PropertyScope.INBOUND);
+        event.getMessage().setProperty("state", "MULE_EVENT_ID=whatever>>", PropertyScope.INBOUND);
```

Comment:
This makes no sense. What's the point of putting that suffix if there's no prefix? It doesn't serve it's purpose. Either replace both suffix and prefix for non conflictive characters or remove both.